indexer/push_new_docker_image.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+TOKEN="$1"
+if [ -z "$TOKEN" ]; then \
+  echo "Usage: $0 GITHUB_TOKEN_FOR_UPLOADING_INDEX"; \
+  exit -1; \
+fi;
+
+TEMP_DIR="$(mktemp -d)"
+# Make sure we delete TEMP_DIR on exit.
+trap "rm -r $TEMP_DIR" EXIT
+
+# Copy all the necessary files for docker image into a temp directory and move
+# into it.
+cp Dockerfile "$TEMP_DIR/"
+cp cronjob "$TEMP_DIR/"
+cp run.sh "$TEMP_DIR/"
+cp ../download_latest_release_assets.py "$TEMP_DIR/"
+cp prepare.sh "$TEMP_DIR/"
+cp entry_point.sh "$TEMP_DIR/"
+cd "$TEMP_DIR"
+
+# Build the image, tag it for GCR and push.
+docker build --build-arg TOKEN="$TOKEN" -t "$IMAGE_IN_GCR" .
```

Comment:
we're not catching failures of these commands, maybe change the shebang to include `-e` (and remove the "bash" from the instructions)

Reply:
got rid of shebang lines and added `set -ex`